third_party/blink/renderer/brave_font_whitelist.h

```diff
@@ -9,7 +9,8 @@
 #include <string>
 #include <string_view>
 
-#include "base/containers/flat_set.h"
+#include "base/containers/fixed_flat_set.h"
+#include "base/no_destructor.h"
```

Comment:
these two headers shouldn't be necessary here.

Reply:
removed